README.md

```diff
@@ -161,15 +162,22 @@ Similar methods exist for BinaryOp, Monoid, and Semiring.
 
 ## Import/Export connectors to the Python ecosystem
 `grblas.io` contains functions for converting to and from:
-- numpy arrays and matrices
```

Comment:
Btw, the numpy matrix is out of style and is basically deprecated.  All mention of it has been removed from this README.